File: update.py
from pymongo import MongoClient
client = MongoClient('localhost:27017')
db = client.TYIT229714
def update():
    try:   
        name = input("Enter Name: ")
        rollno = input("Enter Roll No: ")
        db.Student.update_one(
                                {"name": name},
                                {"$set": {"rollno":rollno}}
        )
        print("Records updated successfully")    
    except Exception:
        print(str(e))

update()

File: update.php
<?php
   // connect to mongodb
   $m = new MongoClient();
   echo "Connection to database successfully";
	
   // select a database
   $db = $m->TYIT_229718;
   echo "Database TYIT_229718 selected";
   $collection = $db->TYITCol;
   echo "Collection selected succsessfully";

   // now update the document
   $collection->update(array("name"=>"Kshamata"), 
      array('$set'=>array("rollno"=>718)));
   echo "Document updated successfully";
	
   // now display the updated document
   $cursor = $collection->find();
	
   // iterate cursor to display title of documents
   echo "Updated document";
	
   foreach ($cursor as $document) {
      echo "<br>".$document["name"] . "<br>";
   }
?>